### PR TITLE
TLS_Cipher_String_Cheat_Sheet: added TLSv1.3

### PR DESCRIPTION
**Pull request by valentinxxx**
This PR covers issue #14.

Added support for `TLSv1.3` using the [default OpenSSL TLSv1.3 ciphersuites](https://www.openssl.org/docs/man1.1.1/man1/ciphers.html) (`TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256`).

### CODE REVIEW

**Comment by righettod** [comment]
Thanks you very much for the PR 😃 
OK for me for the merging.
@ThunderSon @mackowski Your opinion?

File: cheatsheets/TLS_Cipher_String_Cheat_Sheet.md [hunk 1]
```diff
@@ -58,24 +58,27 @@ This table lists the names used by IANA and by openssl in brackets `[]`. Additio
...
-| `TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA`,<br>[`ECDHE-RSA-AES128-SHA`] | 0xc013 |  |  | 10 | 10 |
-| `TLS_RSA_WITH_AES_256_GCM_SHA384`,<br>[`AES256-GCM-SHA384`] | 0x009d |  |  |  | 11 |
-| `TLS_RSA_WITH_AES_128_GCM_SHA256`,<br>[`AES128-GCM-SHA256`] | 0x009c |  |  |  | 12 |
-| `TLS_RSA_WITH_AES_256_CBC_SHA256`,<br>[`AES256-SHA256`] | 0x003d |  |  |  | 13 |
-| `TLS_RSA_WITH_AES_128_CBC_SHA256`,<br>[`AES128-SHA256`] | 0x003c |  |  |  | 14 |
-| `TLS_RSA_WITH_AES_256_CBC_SHA`,<br>[`AES256-SHA`] | 0x0035 |  |  |  | 15 |
-| `TLS_RSA_WITH_AES_128_CBC_SHA`,<br>[`AES128-SHA`] | 0x002f |  |  |  | 16 |
-| `TLS_DHE_RSA_WITH_AES_256_CBC_SHA`,<br>[`DHE-RSA-AES256-SHA`] | 0x0039 |  |  | 11 | 17 |
-| `TLS_DHE_RSA_WITH_AES_128_CBC_SHA`,<br>[`DHE-RSA-AES128-SHA`] | 0x0033 |  |  | 12 | 18 |
+| `TLS_AES_256_GCM_SHA384` *(TLSv1.3)*,<br>[`TLS_AES_256_GCM_SHA384`] | 0x1302 | 1 | 1 | 1 | 1 |
```
Comment (ThunderSon) [inline cheatsheets/TLS_Cipher_String_Cheat_Sheet.md]: I don't believe these priorities still stick as they are, especially after your update. Did you validate the priority vs the following [mozilla URL](https://wiki.mozilla.org/Security/Server_Side_TLS#Cipher_names_correspondence_table)?

Comment (valentinxxx) [reply cheatsheets/TLS_Cipher_String_Cheat_Sheet.md]: Do you mean the existing chipers or the TLSv1.3 ones?
The new ciphers are not present at Mozilla yet, however the existing ones require a refactor if the Mozilla list is the example to follow. I think the sort of priorities is a different issue.

Comment (ThunderSon) [reply cheatsheets/TLS_Cipher_String_Cheat_Sheet.md]: True, let's leave that for another issue. Thank you for the PR.

**Pull request merged**